### DISCUSSION TITLE
Change text color of Tile when entity is negative

### DISCUSSION THREAD

**Discussion by Bellavista** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#243] [Q&A] [answered]
Best card!
Is it possible to change the color of sensor text in red, or the background of button, when it is negative value
![image](https://user-images.githubusercontent.com/46230198/144949165-28645045-cf70-4401-8823-cf46e359ee3b.png)

**Comment by PiotrMachowski** [comment]
It's not possible in a card itself, but it should be doable using [card-mod](https://github.com/thomasloven/lovelace-card-mod)

**Comment by Bellavista** [comment, marked as answer]
Hi Piotr
i' trying someting of simple
card_mod installation work but if i try to use in vacuum-map-card i do someting of wrong
can you help me?
tnk
![image](https://user-images.githubusercontent.com/46230198/147171584-c4717cda-4651-46f6-ac83-de4f606d1cdd.png)

**Reply by PiotrMachowski** [reply]
That's not how it works... Card mod is configured once for the whole card, you have to use selectors to apply styles to a specific element

**Reply by Bellavista** [reply]
done! after two nights ... I arrived at the desired result
![image](https://user-images.githubusercontent.com/46230198/147985845-9768b97b-c060-4948-bfbf-1cf4b88dc639.png)
tnk!
is there a way to write fewer lines
```yaml
  style: |
    {% if (states("sensor.rufus_sensor_dirty_left")) |float(0) < 0 %}
      :host 
        .tile-wrapper.clickable.ripple:nth-child(7) div.tile-value-wrapper div.tile-value 
        {
          color: red;
          font-weight: bold;
        }
        .tile-wrapper.clickable.ripple:nth-child(7) 
        {
          border: 3px solid red;
          border-radius: 10px;
          background : lightgray;
        }
        .tile-wrapper.clickable.ripple:nth-child(7) div.tile-title::before
        {
          content: "Manutenzione ";
        }
    {% endif %};
```
best tnk!

**Reply by PiotrMachowski** [reply]
I think you can reduce it to something following:
```yaml
  style: |
    {% if (states("sensor.rufus_sensor_dirty_left")) |float(0) < 0 %}
      :host 
        .tile-wrapper:nth-child(7).tile-value 
        {
          color: red;
          font-weight: bold;
        }
        .tile-wrapper:nth-child(7) 
        {
          border: 3px solid red;
          border-radius: 10px;
          background : lightgray;
        }
        .tile-wrapper:nth-child(7).tile-title::before
        {
          content: "Manutenzione ";
        }
    {% endif %};
```

**Reply by Bellavista** [reply]
![image](https://user-images.githubusercontent.com/46230198/147990322-a6f84a53-7203-4169-8596-bf63d979c375.png)
all right so! i'm sodisfact of result!
tnk you for your time

**Reply by PiotrMachowski** [reply]
Ok, now the final one:
```yaml
style: |
  {% if (states("sensor.rufus_sensor_dirty_left")) |float(0) < 0 %}
    :host 
      .tile-wrapper:nth-child(7) {
        border: 3px solid red;
      }
      .tile-wrapper:nth-child(7) > * > .tile-value {
        color: red;
        font-weight: bold;
      }
      .tile-wrapper:nth-child(7) > .tile-title::before  {
        content: "Manutenzione ";
      }
  {% endif %}
```

**Reply by Bellavista** [reply]
yesss! it work!
tnk all

**Comment by majdzik84** [comment]
Cool. It will work very well with a single map. With several vacuum cleaners, a certain style is shown in each preset.
Is there any chance to style the tiles for a specific preset?

**Reply by PiotrMachowski** [reply]
@majdzik84 right now it's not possible...
